pkg/venafi/tpp/tpp.go

```diff
@@ -73,7 +73,7 @@ type certificateRequest struct {
 	State                   string          `json:",omitempty"`
 	Country                 string          `json:",omitempty"`
 	SubjectAltNames         []sanItem       `json:",omitempty"`
-	Contact                 string          `json:",omitempty"`
+	Contacts                []contactReq    `json:",omitempty"`
```

Comment:
For some reason, the `Contact` field here was misspelled (should be `Contacts`) and it type was incorrect: it should have been of the form `[{"PrefixedUniversal": "..."}]`. 

That's why I made this change.

Context: https://docs.venafi.com/Docs/currentSDK/TopNav/Content/SDK/WebSDK/r-SDK-POST-Certificates-request.php